packages/features/auth/lib/get-server-session-for-app-dir.ts

```diff
@@ -1,12 +0,0 @@
-import { getServerSession } from "next-auth";
-import { cookies } from "next/headers";
-
-import { getOptions } from "./next-auth-options";
```

Comment:
- this was pulling lots of large dependencies, creating a slow down
- it's better to use our original `getServerSession` function (which has good caching mechanism in place)

Reply:
discussion: https://app.campsite.com/cal/posts/58hm2drix99s